sdk/trace/span_processor_test.go

```diff
@@ -48,10 +42,13 @@ func (t *testSpanProcesor) Shutdown() {
 
 func TestRegisterSpanProcessort(t *testing.T) {
 	name := "Register span processor before span starts"
+	tp, _ := sdktrace.NewProvider(
+		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}))
```

Comment:
This line is repeated a lot.
suggestion:
Set `var testConfig = sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}` and use that in `WithConfig`